_includes/manuals/2.3/1.2_release_notes.md

```diff
@@ -16,6 +16,15 @@ When template safe mode rendering is disabled in the settings, users can now pre
 #### Show instance name in top menu
 For users managing multiple Foreman instances, a new setting (`instance_title`) has been introduced that allows setting a name for each instance. When set, an icon will appear on the top bar which will display the instance name when hovered over. ([#29024](https://projects.theforeman.org/issues/29024))
 
+### Release notes for 2.3.5
+#### Installer - Foreman modules
+* smart_proxy_dynflow_core is not restarted after re-configuration ([#32078](https://projects.theforeman.org/issues/32078))
+
+#### SELinux
+* Denial when installing on CentOS 8 Stream ([#32023](https://projects.theforeman.org/issues/32023))
```

Comment:
I wonder if by now we should change this to EL 8.4. It also affects CentOS Linux 8.4 and RHEL 8.4. Perhaps a general (handwritten) note that this is the last release that's just for compatibility allowing users to upgrade to 2.4?

Reply:
Good idea!

Reply:
Updated, request to check. Thank you!